5-infrastructure/modules/base-env/variables.tf

```diff
@@ -0,0 +1,295 @@
+  description = "The location of the second GKE cluster.  Only if Secondary region value, is not given"
+  default     = "us-west1"
+}
+
+variable "gke_cluster_2_cidr_block" {
+  type        = string
+  description = "The primary IPv4 cidr block for the second GKE cluster."
+}
+
+variable "gke_cluster_2_subnet_name" {
+  type        = string
+  description = "The name of the subnet for the second GKE cluster."
+  default     = "gke-cluster2-subnet"
+}
+
+variable "gke_cluster_2_range_name_pods" {
+  type        = string
+  description = "The name of the pods IP range for the second GKE cluster."
+  default     = "pod-ip-range"
+}
+
+variable "gke_cluster_2_range_name_services" {
+  type        = string
+  description = "The name of the services IP range for the second GKE cluster."
+  default     = "services-ip-range"
+}
+
+variable "gke_cluster_2_machine_type" {
+  type        = string
+  description = "The type of VM that will be used for the second GKE cluster (ex. e2-micro)."
+  default     = "e2-standard-4"
+}
+
+variable "gke_mci_cluster_location" {
+  type        = string
+  description = "The location for multi-cluster ingress (MCI). Only if primary region value, is not given"
+  default     = "us-east1"
+}
+
+variable "gke_mci_cluster_cidr_block" {
+  type        = string
+  description = "The primary IPv4 cidr block for multi-cluster ingress (MCI)."
+}
+
+variable "gke_mci_cluster_subnet_name" {
+  type        = string
+  description = "The name of the subnet for multi-cluster ingress (MCI)."
+  default     = "mci-config-subnet"
+}
+
+variable "gke_mci_cluster_range_name_pods" {
+  type        = string
+  description = "The name of the pods IP range for multi-cluster ingress (MCI)."
+  default     = "pod-ip-range"
+}
+
+variable "gke_mci_cluster_range_name_services" {
+  type        = string
+  description = "The name of the services IP range for multi-cluster ingress (MCI)."
+  default     = "services-ip-range"
+}
+
+variable "gke_mci_cluster_machine_type" {
+  type        = string
+  description = "The type of VM that will be used for multi-cluster ingress (MCI)."
+  default     = "e2-standard-2"
+}
+
+variable "boa_gke_project_id" {
+  type        = string
+  description = "Project ID for GKE."
+}
+
+variable "boa_ops_project_id" {
+  type        = string
+  description = "Project ID for ops."
+}
+
+variable "boa_sec_project_id" {
+  type        = string
+  description = "Project ID for secrets."
+}
+
+variable "boa_sql_project_id" {
+  type        = string
+  description = "Project ID for SQL."
+}
+
+variable "sql_1_database_region" {
+  type        = string
+  description = "The location of the SQL database. Only if primary region value, is not given"
+  default     = "us-east1"
+}
+
+variable "sql_2_database_region" {
+  type        = string
+  description = "The location of the 2nd SQL database. Only if Secondary region value, is not given"
+  default     = "us-west1"
+}
+
+variable "sql_instance_defaults" {
```

Comment:
Lets simplify this and require only one set of username/passwords, use the two regions we know of from primary and secondary location variables